### DISCUSSION TITLE
[node] zlib types aren't correct when { info: true } is passed as option

### DISCUSSION THREAD

**Discussion by magcius** [DefinitelyTyped/DefinitelyTyped#68138] [Issues with a @types package]
According to [the documentation](https://nodejs.org/api/zlib.html#class-options):

> info [<boolean>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type) (If true, returns an object with buffer and engine.)

However, the type description for e.g. inflateSync seems to imply that it will always return a Buffer type. An override is necessary here to add support for the `info: true` case.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
Related PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67291
